### ISSUE TITLE
Solo Hotel:

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#112]
copied from original to-do list in code:
```
- Front office girl is stuck and has to pee
- Room service girl brings & shares champagne
- Escort in-call
```
Probably not in line with the goal of the game but documenting here that it exists.

**Comment by M0nkey5ee** [comment]
Actually a great idea! Would love to build it with ya :]

**Comment by Forget-About-Me2** [comment]
Not sure a way to dump the girl should be added, though the idea to take the girl to a hotel could be considered with some features might be an option to branch out.

**Comment by M0nkey5ee** [comment]
True. Maybe a jacuzzi-party with 2-3 girls all at once? A hotelbar? A fitness-studio in the hotel? Then taking a room, given enough money?
